CONTRIBUTING_DOCS.md

```diff
@@ -93,57 +91,77 @@ Let's say there's a single commit that makes changes to the `master` directory w
+This page describes Calico and the primary reasons for using it.
 
-### Getting started
+### Install
 
-This should be where new users go. It includes quick-start guides, some basic tutorials to show off Calico's capabilities, and links to more advanced topics once users are comfortable with the basics.
+Calico can be installed in many types of deployments including on-premises and in cloud providers. This section includes how to install a standalone Calico cluster for workloads, and how to install Calico on non-cluster hosts. This section reflects steps to an "up and running" state. Every task beyond "up and running" should be added to other tabs. 
 
-Each orchestrator has a landing page that is targeted at people who are coming to see Calico for the first time. It's a transition from the "marketing" type material (why is Calico great) to some quick commands people can run to see it firsthand, and then funnels people off to the Usage section for more details.
+### Operations
 
-### Usage
+This section contains post-install, task-based content.  
 
-This section contains task-based information. All top-level titles in this section should start with a gerund. Each topic should include why you want to perform the task, a goal, and a set of steps you can follow to achieve it.
+### Networking
 
-Examples:
+This section contains task-based content for networking using the Calico CNI and Calico IPAM.
 
-- Configuring BGP peers
-- Enabling IP-in-IP in AWS
-- Troubleshooting Calico
-- Using calicoctl in a Kubernetes deployment
-- Configuring egress policy in Kubernetes
+### Security
 
-Do not include detailed description of components or tabulated
-configuration information in this section. This type of content should be located in the [Reference](#reference) section.
+This section contains task-based content for securing Calico components, workloads, and non-cluster hosts using Calico network policy.
 
 ### Reference
 
-These docs contain complete reference information for Calico. If there's a configuration option you're looking for, it goes here in one of the per-component references. Not every option has a "how to" guide, but has enough description. The caveats and considerations when enabling options should be listed here.
+This section contain reference content including APIs and Resources. Add configuration options in one of the per-component references, and list any caveats and considerations when enabling options.
 
 Examples:
 
 - Fully tabulated configuration options per-component
 - `calicoctl` help text
 - Calico API schema reference (policy, ip pool, etcd)
 
-
 ## Organizational changes
 
 ### Creating new pages
 
-- To create a top level splash page for a URL path, simply name the file `index.md`.
+- To create a top level splash page for a URL path, name the file `index.md`. 
+
+  If the index.md has child topics, add the following. The `description` field must be lowercase or you will get an error during the build. For example:
```

Comment:
Err.. description in the following example is evidently not lowercase, so I don't understand what you mean here.

Reply:
I think it means the field name/key, not the value.  Perhaps changing "field" to "field name/key" would make this clearer.  But perhaps we should just say all of the field names/keys should be lower case for consistency (even if they don't break the build) and don't mention breaking the build.

Reply:
Note that Reza is working on a final version of the tabs incantation which supports linking multiple tab groups.  So for example if you are choosing between "Operator" and "Manifest" earlier in the doc, then a later tab section in the same doc will default to showing which ever tab you selected earlier.  So you might want to put "details coming soon" for the tabs and ask Reza to do a small PR when it's all finalized.

Reply:
Clarified.

Reply:
Fixed to name/key per new comment.